run_kraken.py

```diff
@@ -49,6 +49,7 @@ def main(cfg):
         run_signal = config["kraken"].get("signal_state", "RUN")
         litmus_version = config["kraken"].get("litmus_version", "v1.9.1")
         litmus_uninstall = config["kraken"].get("litmus_uninstall", False)
+        litmus_uninstall_before_run = config["kraken"].get("litmus_uninstall_before_run", True)
```

Comment:
We might want to expose this option in the config @aaguirreRH : https://github.com/chaos-kubox/krkn/blob/main/config/config.yaml and https://github.com/chaos-kubox/krkn/blob/main/config/config_performance.yaml.

Reply:
Done!